frontend/micro-ui/web/micro-ui-internals/packages/modules/ws/src/utils/getWSAcknowledgementData.js

```diff
@@ -46,7 +46,7 @@ const getPropertyAddress = (property,t) => {
   const street = property?.street;
   const landMark = property?.landmark;
   const locality = property?.locality?.name;
-  const city = t(`TENANT_TENANTS_${property?.tenantId?.toUpperCase?.()?.replace(".", "_")}`);
```

Comment:
@Hari-egov Kindly review this, this may impact the localisation data what we have loaded alread